04/index.html

```diff
@@ -1,30 +1,94 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
-                </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
+
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/global.css" />
+    <link
+      rel="stylesheet"
+      media="(max-width: 760px)"
+      href="./styles/mobile.css"
+    />
+    <link
+      rel="stylesheet"
+      media="(min-width: 761px)"
+      href="./styles/tablet&desktop.css"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="container__nav">
+        <input type="checkbox" class="hamburger" />
+
+        <ul class="menu nav">
+          <li class="nav__item red-border">
+            <a href="index.html">START</a>
+          </li>
+
+          <li class="nav__item">
+            <label for="submenu-switcher">OFERTA</label>
+            <input type="checkbox" id="submenu-switcher" />
+
+            <ul class="menu2 nav__dropdown nav__dropdown--lvl1">
+              <li class="dropdown__item-lvl1 red-border">
```

Comment:
Tutaj podobnie jak wyżęj. Jeśli mamy element `dropdown__item-lvl1`, to gdzieś wyżej (pewnie w `ul`) powinien pojawić się blok tj. `dropdown`